src/Uno.UI/Microsoft/UI/Xaml/Controls/MonochromaticOverlayPresenter/MonochromaticOverlayPresenter.cs

```diff
@@ -45,68 +45,68 @@ private void InvalidateBrush()
 
 		private void UpdateBrush()
 		{
-			if (SourceElement is { } targetElement)
-			{
-				var newColor = ReplacementColor;
-				if (_replacementColor != newColor)
-				{
-					_replacementColor = newColor;
-					_effectFactory = null;
-				}
+			//if (SourceElement is { } targetElement)
```

Comment:
@MartinZikmund we did not need this at all ?

Reply:
It didn't work actually, and threw an exception which was masked, so I instead adjusted the styles so it is not needed. But will ad a runtime check instead to keep the code in